src/resolvers.ts

```diff
@@ -115,10 +116,10 @@ export async function getResolvers(
   }
 
   // Compute the content hash.
-  for (const f of assets) hash.update(loaders.getFileHash(resolvePath(path, f)));
-  for (const f of files) hash.update(loaders.getFileHash(resolvePath(path, f)));
+  for (const f of assets) hash.update(loaders.getPreviewFileHash(resolvePath(path, f)));
+  for (const f of files) hash.update(loaders.getPreviewFileHash(resolvePath(path, f)));
   for (const i of localImports) hash.update(getModuleHash(root, resolvePath(path, i)));
-  if (page.style && isPathImport(page.style)) hash.update(loaders.getFileHash(resolvePath(path, page.style)));
+  if (page.style && isPathImport(page.style)) hash.update(loaders.getPreviewFileHash(resolvePath(path, page.style)));
```

Comment:
This is also confusing since of course resolvers are used during build. But the content hash is only used during preview, so this likewise doesn’t affect the build.